### PR TITLE
Update index.rst

### PR DESCRIPTION
**Pull request by souravsingh**
Added a new line mentioning the link to sunpy.org

### CODE REVIEW

File: doc/source/index.rst [hunk 1]
```diff
@@ -15,3 +15,6 @@ free and open-source solar data analysis environment for Python.
  
 .. warning::
     Having issues with links on this page? Make sure there is a trailing slash on the URL *e.g.* `http://docs.sunpy.org/en/latest/`
+
+-Link to SunPy
+    Want to visit the blog and stay updated with the tutorials?Make sure there is a trailing slash on the URL *e.g.* `http://sunpy.org/`
```
Comment (dpshelio) [inline doc/source/index.rst]: What's the problem with the trailing slash? it works for me without it.
Also, Could you add an space after the question mark?

Thanks for contributing :)

Comment (Cadair) [inline doc/source/index.rst]: I don't think this is the best place for this, it should be at the top of the page, this warning about the trailing slash was to work around an old bug in read the docs, which now fixed.

Comment (souravsingh) [inline doc/source/index.rst]: Alright.I will make the required changes.

Comment (Cadair) [inline doc/source/index.rst]: thanks.

Comment (souravsingh) [inline doc/source/index.rst]: Since the bug about the trailing slash is fixed.Would it be okay if I remove it from the RST file?

**Comment by souravsingh** [comment]
Can anyone tell me about the reason for failures of build tests? Is it something to do with the changes I made?

**Comment by Cadair** [comment]
@souravsingh I have done some fixes in master to update the CI builds. Can you merge master into this?

**Pull request closed without merging**